### PR TITLE
Issue 104 Add ResponsibilityRoles to RESOURCE FRAME

### PR DESCRIPTION
**Pull request by nick-knowles**
No way of adding user defined RESPONSIBILITY ROLES.  - element added to ResourceFRame

### CODE REVIEW

File: xsd/netex_framework/netex_frames/netex_resourceFrame_version.xsd [hunk 1]
```diff
@@ -124,6 +127,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Definitions of DATA SOURCE included in frame.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="responsibilityRoles" type="responsibilityRolesInFrame_RelStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>RESPONSIBILITY Roles used in frame.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_frames/netex_resourceFrame_version.xsd:132]: ROLES in caps

Comment (ue71603) [reply xsd/netex_framework/netex_frames/netex_resourceFrame_version.xsd]: changed it.

**Review by ue71603 — APPROVED** [review]

File: xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd [hunk 2]
```diff
@@ -22,6 +22,10 @@
 					<Modified>2017-04-27   CR0038 Add  REsponsibilityRole as Abstract entity , add general Role </Modified>
 					<Modified>2017-04-27   CR0038 Add  PrivateCOde to ResponsibilitySet </Modified>
 				</Date>
+
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_responsibilitySet_version.xsd:25]:
```suggestion
```

**Review by skinkie — APPROVED** [review]

**Comment by skinkie** [comment]
@ue71603 if you see the files changed, it was already in there. I leave the capital change at this pull request, that can be fixed later.

**Pull request closed without merging**